cmake/modules/BuildDPDK.cmake

```diff
@@ -118,9 +118,8 @@ function(do_export_dpdk dpdk_dir)
 
   list(APPEND dpdk_components
     bus_pci
-    cmdline
+    telemetry
```

Comment:
Nit: can we keep this in alphabetical order?

Reply:
Sure